src/Xamarin.Android.Build.Tasks/Utilities/ApplicationConfigTaskState.cs

```diff
@@ -0,0 +1,9 @@
+namespace Xamarin.Android.Tasks
+{
+	class ApplicationConfigTaskState
+	{
+		public const uint RegisterTaskObjectKey = 0xDEADBEEF;
```

Comment:
If this was a `string` would it be more readable? If they key was `Xamarin.Android.Tasks.ApplicationConfigTaskState` or something?

I just imagine debugging and trying to figure out what `0xDEADBEEF` means.